### ISSUE TITLE
[Accessibility]: Code monitoring ARC checklist issues

### ISSUE THREAD

**Issue by rvantonder** [sourcegraph/sourcegraph#35155]
- H1 `Code Monitoring` header needs to be H2.

- Image issue on SVGs
  - `missing role = 'img'`
  - needs `aria label`

See screenie:
<img width="695" alt="Screen Shot 2022-05-09 at 15 53 44" src="https://user-images.githubusercontent.com/12631702/167487663-26c187d3-c51a-44cd-af49-9d601b9d0fc0.png">

**Comment by sourcegraph-bot-2** [comment]
Heads up @benvenker @lguychard - the "team/search-product" label was applied to this issue.
